### DISCUSSION TITLE
Contributions が一定値に達した人へのご褒美

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#34] [要望・アイデア・不具合]
Twitter だと誕生日に特殊な演出が行われたりしますが、Qiita で Contributions が 100 とか 1000 を超えたときに画面上にそれを祝うアニメーションやプロフィール画面に演出を出したりとか（Youtube の銀の盾、金の盾みたいな）、Contributions が特にすごい人のプロフィール画像には後光がついて表示されたりとかしてみるのはどうでしょう？

**Comment by kskwtnk** [comment]
@blono 

ご意見ありがとうございます！

Contributionが○○に達した際のアニメーション……などは現状ないのですが、YouTubeの銀の盾金の盾のような制度が最近スタートしました！
https://qiita.com/qiita-award

こちらは本当に最近スタートしたばかりで全然広められていないので今後は広めて行く予定です！

また、アニメーションなどの演出も議論にはあがっているため、表現を考えていきます！

**Reply by uhooi** [reply]
おわぁ表彰知りませんでした…！
シルバー狙って行きたいと思います💪

**Reply by kskwtnk** [reply]
@uhooi ありがとうございます！ぜひぜひ狙ってください！！
